src/components/RadioButton/radio.module.scss

```diff
@@ -23,13 +23,14 @@
 
         & + label {
             .radio-button {
```

Comment:
We should work to avoid deeply nesting css selectors when possibly to reduce specificity and css output